### PR TITLE
feature/#129

### PR DESCRIPTION
**Pull request by DenisKulik**

### CODE REVIEW

File: src/features/profile/ui/personalInformation/avatarUploader/AvatarUploader.tsx [hunk 1]
```diff
@@ -1,29 +1,50 @@
...
   name,
   editable = true,
   className,
+  updateAvatar,
 }: Props): JSX.Element => {
   const avatarUploaderClasses = cn(s.root, className)
 
+  const [image, setImage] = useState(avatar)
+
+  useEffect(() => {
```
Comment (vitalyabedik) [inline src/features/profile/ui/personalInformation/avatarUploader/AvatarUploader.tsx:29]: try do without useEffect

File: src/pages/profilePage/ProfilePage.tsx [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
-import { PersonalInformation } from '@/features'
+import { Route } from '@/common'
+import { GoBack, Page } from '@/components'
+import {
+  PersonalInformation,
+  ProfileDataType,
+  useMeQuery,
+  useUpdateProfileMutation,
+} from '@/features'
+import { EditProfileValues } from '@features/profile/ui/personalInformation/editProfile/useEditProfile.ts'
```
Comment (vitalyabedik) [inline src/pages/profilePage/ProfilePage.tsx:9]: fix import

File: src/features/profile/ui/personalInformation/avatarUploader/AvatarUploader.tsx (hunk 1 above)
Comment (DenisKulik) [reply src/features/profile/ui/personalInformation/avatarUploader/AvatarUploader.tsx]: Done!

File: src/pages/profilePage/ProfilePage.tsx (hunk 2 above)
Comment (DenisKulik) [reply src/pages/profilePage/ProfilePage.tsx]: Done!

File: src/features/auth/api/authApi.types.ts [hunk 3]
```diff
@@ -30,3 +30,5 @@ export type LoginParamsType = SignUpDataType & {
 export type RecoverPasswordParamsType = Pick<SignUpParamsType, 'email'>
 
 export type ResetPasswordParamsType = Pick<SignUpParamsType, 'password'> & { token: string }
+
+export type UpdateProfileFormDataType = FormData
```
Comment (vitalyabedik) [inline src/features/auth/api/authApi.types.ts:34]: maybe without it

File: src/features/profile/ui/personalInformation/avatarUploader/AvatarUploader.module.scss [hunk 4]
```diff
@@ -3,18 +3,21 @@
...
     bottom: 0;
+    text-align: right;
 
-    background-color: var(--color-dark-300);
-    border-radius: 4px;
-    box-shadow: 0 2px 10px 0 rgb(109 109 109 / 25%);
+    .editAvatar {
+      background-color: var(--color-dark-300);
+      border-radius: 4px;
+      box-shadow: 0 2px 10px 0 rgb(109 109 109 / 25%);
```
Comment (vitalyabedik) [inline src/features/profile/ui/personalInformation/avatarUploader/AvatarUploader.module.scss:15]: rem or not?

File: src/features/auth/api/authApi.types.ts (hunk 3 above)
Comment (DenisKulik) [reply src/features/auth/api/authApi.types.ts]: done!

File: src/features/profile/ui/personalInformation/avatarUploader/AvatarUploader.module.scss (hunk 4 above)
Comment (DenisKulik) [reply src/features/profile/ui/personalInformation/avatarUploader/AvatarUploader.module.scss:15]: done!

**Review by vitalyabedik — APPROVED** [review]
Good!

**Review by VitaliKozhurov — APPROVED** [review]
Good!

**Pull request merged**